FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -359,16 +360,32 @@ - (NSString *)nextRequestWithUserProperties:(NSDictionary *)userProperties {
 
   if (userProperties && userProperties.count > 0) {
     NSError *error;
-    NSData *jsonData = [NSJSONSerialization dataWithJSONObject:userProperties
-                                                       options:0
-                                                         error:&error];
-    if (!error) {
-      ret = [ret
-          stringByAppendingString:[NSString
-                                      stringWithFormat:@", analytics_user_properties:%@",
-                                                       [[NSString alloc]
-                                                           initWithData:jsonData
-                                                               encoding:NSUTF8StringEncoding]]];
+
+    // Extract first open time from user properties and send as a separate field
+    NSNumber *firstOpenTime = [userProperties valueForKey:kRCNAnlyticsFirstOpenTimePropertyName];
+    if (firstOpenTime) {
+      NSDate *date = [NSDate dateWithTimeIntervalSince1970:([firstOpenTime longValue] / 1000)];
+      NSISO8601DateFormatter *formatter = [[NSISO8601DateFormatter alloc] init];
+      NSString *firstOpenTimeISOString = [formatter stringFromDate:date];
+      ret = [ret stringByAppendingString:[NSString stringWithFormat:@", first_open_time:'%@'",
+                                                                    firstOpenTimeISOString]];
+
+      NSMutableDictionary *userPropertiesCopy = [userProperties mutableCopy];
+      [userPropertiesCopy removeObjectForKey:kRCNAnlyticsFirstOpenTimePropertyName];
```

Comment:
Might be a bit cleaner to do initialize something like `NSMutableDictionary *remainingUserProperties = [userProperties mutableCopy]` before line 365.